frontend/src/components/RecommendationTagElement.js

```diff
@@ -0,0 +1,57 @@
+import styled from 'styled-components/macro';
+import { Multiselect } from 'multiselect-react-dropdown';
+import {useEffect, useRef} from 'react';
+
+
+export default function RecommendationTagElement ({recommendationTagObject, getRecommendation, setRecommendation}){
+
+    const recommendationValueRef = useRef();
+
+    const selectionLimit =
+        recommendationTagObject.categoryName === "Gender" ? 1 :
+        recommendationTagObject.categoryName === "Roles" ? 2 : 3;
```

Comment:
😓 nested ternaries are hard to read, don't force me to think when reading your code :)

Reply:
you got a point. I reformatted to make it more clear. I prefer this over if else statements...